x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/common_attributes.schema.yaml

```diff
@@ -42,6 +42,52 @@ components:
 
     IsRuleImmutable:
       type: boolean
+      deprecated: true
+      description: This field determines whether the rule is a prebuilt Elastic rule. It will be replaced with the `rule_source` field.
+
+    IsExternalRuleCustomized:
+      type: boolean
+      description: Determines whether an external/prebuilt rule has been customized by the user (i.e. any of its fields have been modified and diverged from the base value).
+
+    ExternalSourceUpdatedAt:
```

Comment:
The initial use case for this field (last time the rule was created/updated by the TRaDE team) was put on hold from the implementation side on the TRaDE team, meaning that at least for the time being we won't have input data in the `security-asset`s to populate this field with.

Let's remove this field until we have more clarity of when or if it is going to be used at all.

Reply:
Okay, removed it altogether